model_fitting/scan_peptides_vs_motifs.py

```diff
@@ -16,20 +16,23 @@
 
 
 def calculate_pssm_thresholds(meme_path, cutoffs_path, faa_path, number_of_random_pssms, 
-                              shuffles, shuffles_percent, shuffles_digits, output_path, done_path, rank_method, argv='no_argv',
+                              shuffles, shuffles_percent, shuffles_digits, use_factor, output_path, done_path, rank_method, argv='no_argv',
                               pssm_score_peptide='./PSSM_score_Peptide/PSSM_score_Peptide'):
 
     if not os.path.exists(output_path):
         # TODO: any modules to load?
         if rank_method == 'pval':
             cmd = f'{pssm_score_peptide} -pssm {meme_path} -pssm_cutoffs {cutoffs_path} -seq {faa_path} ' \
-                f'-out {output_path} -NrandPSSM {number_of_random_pssms} -CalcPSSM_Pval'
+                f'-out {output_path} -NrandPSSM {number_of_random_pssms} -useFactor {use_factor} -CalcPSSM_Pval'
```

Comment:
this outputs `True` of `False` which are not in C++ notation, it is better to use the flag without any values.
i.e., if the flag is defined then use factor

Reply:
This is a bit problematic because it will change the order with the variable CalcPSSM_Pval so I sent 0 or 1 instead.

Reply:
Order doesn't matter. Review the code, it iterates over all arguments without expected order